pom.xml

```diff
@@ -172,6 +173,17 @@
         <module>testing/trino-tests</module>
     </modules>
 
+    <repositories>
```

Comment:
There shouldn't be any references to other repositories than maven central. Otherwise, it can be hard or problematic to build the project in environments that sit behind firewalls.

Reply:
I'm not sure if we can get rid of this dependency. The serde has shims for HDP and CDH, both of which have their own repos. I'm not sure if it will be possible to generalize it to only dependencies found on Maven Central.

Reply:
Turns out this isn't too difficult: https://github.com/starburstdata/hive-json-serde/pull/6

Reply:
cc @electrum